src/validations/access-checker/profiles.ts

```diff
@@ -82,7 +82,9 @@ async function allEmoteUrns(entity: Entity) {
   for (const avatar of allAvatars) {
     const allEmotes = avatar.avatar.emotes ?? []
     for (const { slot, urn } of allEmotes) {
-      allEmotesInProfilePromises.push(sanitizeUrn(urn))
+      if (!isOldEmote(urn)) {
```

Comment:
can we document somewhere the decision of keeping old emotes separated from emotes?

Reply:
This is for *allowing* `oldEmotes` in `emotes`